content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -173,11 +172,11 @@ by default [RBAC core components roles](/docs/reference/access-authn-authz/rbac/
 - A kubeconfig file for scheduler, `/etc/kubernetes/scheduler.conf`; inside this file is embedded a client certificate with scheduler identity.
   This client cert should have the CN `system:kube-scheduler`, as defined by default [RBAC core components roles](/docs/reference/access-authn-authz/rbac/#core-component-roles)
 
-Additionally, a kubeconfig file for kubeadm to use itself and the admin is generated and save into the `/etc/kubernetes/admin.conf` file.
-The "admin" here is defined the actual person(s) that is administering the cluster and want to have full control (**root**) over the cluster.
+Additionally, a kubeconfig file for kubeadm itself and the admin is generated and saved into the `/etc/kubernetes/admin.conf` file.
+The "admin" here is defined as the actual person(s) that is administering the cluster and wants to have full control (**root**) over the cluster.
 The embedded client certificate for admin should:
-- Be in the `system:masters` organization, as defined by default [RBAC user facing role bindings](/docs/reference/access-authn-authz/rbac/#user-facing-roles)
-- Include a CN, but that can be anything. Kubeadm uses the `kubernetes-admin` CN
+  - Be in the `system:masters` organization, as defined by default [RBAC user facing role bindings](/docs/reference/access-authn-authz/rbac/#user-facing-roles)
```

Comment:
the formatting is off here. Not sure how to fix it.

Reply:
it looks like their might be a space before `- Be in`? Is that what you're talking about?

Reply:
Yes, I want that to render as a list. However, it is being rendered as part of the paragraph.

Reply:
i think you can try spacing `- Be` one extra line bellow `- Include`...
there might be markdown bugs at play.

Reply:
FWIW, it renders as a bulleted list on my local.

Reply:
that is the goal, correct?

Reply:
yes, that is the goal. On the netlify link, it is being rendered as part of the paragraph. But on my localhost, it shows as a list.

Reply:
How are you running local previews @prasadkatti?

You may need to update Hugo (or maybe downgrade?). You can also build a container image for local development that includes the right version of Hugo.

Reply:
My Hugo version matches the one that minikube uses. I will try to run it using the container and see how that goes. The Hugo version I have `v0.68.3`. This project is using `v0.59.1`. So yeah, that could be the reason.

Reply:
alternatively, just combine them as sentences in the above paragraph, if the k/website hugo does not render this correctly.

Reply:
I could, but that would be inconsistent with all the other places where we have this as a list. I think we can just leave it like that for now. When Hugo is upgraded this will get rendered the way we want it.
I ran this locally using the container image and it doesn't render properly with Hugo v0.59.1.

Reply:
i'd prefer if we get this working now instead.
mostly because i don't know what is the timeline for the hugo upgrade and people might see this and try to send addition issues / prs.

Reply:
sounds good. In that case, I will just fold the list into the paragraph as you suggested. It will be inconsistent with the other places where we have similar lists as previously mentioned.
Will push up the commit shortly.

Reply:
Done.

Reply:
/lgtm
thanks